app/dao/services_dao.py

```diff
@@ -429,8 +429,12 @@ def dao_fetch_stats_for_service(service_id, limit_days):
 
 
 def dao_fetch_todays_stats_for_service(service_id):
+    today = date.today()
+    start_date = get_london_midnight_in_utc(today)
+    end_date = get_london_midnight_in_utc(today + timedelta(days=1))
     return _stats_for_service_query(service_id).filter(
-        func.date(Notification.created_at) == date.today()
+       Notification.created_at >= start_date,
+       Notification.created_at < end_date
```

Comment:
weird indentation here, this should be one space further forward please

Reply:
also do we even need the end_date at all since we know we're explicitly talking about today?